torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -140,7 +140,8 @@ def generate_ttir(kernel, kwargs):
         if isinstance(a, (torch.SymInt, torch.SymFloat, torch.SymBool)):
             ordered_args[name] = 2
         elif isinstance(a, FakeTensor):
-            ordered_args[name] = torch.empty(2, dtype=a.dtype)
+            with torch._C._DisableTorchDispatch():
+                ordered_args[name] = torch.empty(2, dtype=a.dtype, device=a.device)
```

Comment:
I don't know if we actually need to return a CUDA Tensor here. I can try on a CPU Tensor and see what happens if matters

Reply:
My previous code did not return a cuda tensor, so perhaps not needed. After all, this code will not get executed it is only to generate metadata at triton side.

Reply:
Okay, let's try CPU